utils/publish-canary.js

```diff
@@ -14,7 +15,9 @@ const {
 
 const isPrerelease = TRAVIS_BRANCH.match(/^prerelease\/v\d*$/g);
 const data = {};
+
 let distTag;
+let preid;
```

Comment:
Choosing to update `preid` as a side-effect here. If there's any other way we want to pass `preid` around to each promise callback (like promise inside a promise so the nested one has the parent's scope), I'm all ears.